src/pages/components/pagination.mdx

```diff
@@ -41,6 +41,18 @@ import CursorDefaultCode from '!!raw-loader!../../examples/pagination/CursorDefa
   <CursorDefault />
 </CodeExample>
 
+### Offset pagination
+
+Offset-based pagination allows users to see the total number of pages and jump
+to a specific position within the set.
+
+import OffsetDefault from '../../examples/pagination/OffsetDefault.tsx';
+import OffsetDefaultCode from '!!raw-loader!../../examples/pagination/OffsetDefault.tsx';
+
+<CodeExample code={OffsetDefaultCode}>
+  <OffsetDefault />
+</CodeExample>
+
```

Comment:
Minor change since the next two sections follow configurations of this component. If that seems incorrect we can reverse course.

Reply:
Hmm, originally component pages were structured to alphabetize the examples. Although that approach has been challenged (with Draggable, for example) my instinct is to just leave as-is until the website's content design receives a more thorough overhaul.